pkg/cluster/internal/create/actions/config/config.go

```diff
@@ -185,8 +185,8 @@ func (a *Action) Execute(ctx *actions.ActionContext) error {
 					return errors.Wrap(err, "failed to write patched containerd config")
 				}
 				// restart containerd now that we've re-configured it
-				// skip if the systemd (also the containerd) is not running
-				if err := node.Command("bash", "-c", `! systemctl is-system-running || systemctl restart containerd`).Run(); err != nil {
+				// wait until boot has finished
```

Comment:
we want to only restart it if we observed it was already running after completing the patching, if it wasn't running waiting and restarting it is:
a) slow
b) unnecessary (as it never read the old unpatched config)

there's still a race as to wether or not it starts before we check to restart it, but if we check to restart it and it wasn't running we're fine and if it was running we restart it and wether or not it read the old config it's fine and we don't waste much time.

the problem previously is an ineffective check for it it is running.